extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -683,32 +683,6 @@ public function getMessagePageUrl( $withoutAnchor = false, $fullUrl = true ) {
 		return $this->messagePageUrl[ $withoutAnchor ];
 	}
 
-	public function getArticleId( &$articleData = null ) {
-		$title = $this->getArticleComment()->getTitle();
-		$articleId = $this->getArticleComment()->getTitle()->getArticleId();
-
-		if ( $articleId === false ) {
-			Wikia::log( __METHOD__, false, "WALL_NO_ARTILE_ID" . print_r( [ '$title' => $title ], true ) );
```

Comment:
In Kibana there are no log entries of this, and this method was only used in 2 places: in builder and in RC hook handler.